README.md

```diff
@@ -46,6 +46,7 @@ Make sure Docker can be executed without sudo for the scripts in this guide to w
     sudo groupadd docker
     sudo usermod -aG docker $(id -un)
     sudo systemctl restart docker
+    newgrp docker
```

Comment:
This will only change the group of the logged in user. After logout+login the default user group will be restored. Adding the user to the group of docker should be enough. Could you please check the owner and permissions of the /var/run/docker.sock on your Ubuntu installations ?

Reply:
The logout and login step worked this time. For me, the earlier documentation wasn't clear so I updated the instructions with your feedback. Please review.